apps/tiff2geotiff/tiff2geotiff.cpp

```diff
@@ -99,12 +99,12 @@ extern int         GTIFSetFromProj4_WRF(GTIF *gtif, const char *proj4);
 
 int main(int argc, char *argv[])
 {
-    uint16             defconfig = (uint16)-1;
-    uint16             deffillorder = 0;
-    uint32             deftilewidth = (uint32)-1;
-    uint32             deftilelength = (uint32)-1;
-    uint32             defrowsperstrip = (uint32)-1;
-    uint32             diroff = 0;
+    uint16_t             defconfig = (uint16_t)-1;
+    uint16_t             deffillorder = 0;
+    uint32_t             deftilewidth = (uint32_t)-1;
+    uint32_t             deftilelength = (uint32_t)-1;
+    uint32_t             defrowsperstrip = (uint32_t)-1;
+    uint32_t             diroff = 0;
```

Comment:
You could simply typedef uint32.

Reply:
As I understand it, uint32_t is standard.  Some compilers `typedef uint32 uint32_t;` and it's discouraged to typedef it in user code.  Wouldn't typedefing it bring the potential for unnecessary confusion, versus just using the actual naming standard?